packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -191,6 +198,15 @@ export const GiantTrackTile = ({
   )
   const isUSDCPurchaseGated = isContentUSDCPurchaseGated(streamConditions)
   const isEditAlbumsEnabled = getFeatureEnabled(FeatureFlags.EDIT_ALBUMS)
+  const isLosslessDownloadsEnabled = getFeatureEnabled(
+    FeatureFlags.LOSSLESS_DOWNLOADS_ENABLED
+  )
+  const track = useSelector(
+    (state: CommonState) => getTrack(state, { id: trackId }),
+    shallowEqual
+  )
+  // TODO: also check has_stems when that is added PAY-2347
+  const hasDownloadableAssets = track?.is_downloadable // || track?.has_stems
```

Comment:
has_stems won't be added as per our discussion earlier. we should be able to grab the stems from the track still no? maybe the `_stems` is set, or the `useCurrentStems`. not sure exactly how but the page does know if track has stems

Reply:
my b i thought we said we were going to add it but i guess that was just a helper var during indexing validation!